.github/workflows/config/spelling_allowlist.txt

```diff
@@ -198,10 +202,13 @@ lossy
 lvalue
 makefiles
 merchantability
+mlir
```

Comment:
Suggest to use `MLIR` it's already in the list.

Reply:
Fixed in [a3964e2](https://github.com/NVIDIA/cuda-quantum/pull/1763/commits/a3964e284910f99cbbddddfa7493747ab9735855).